api/_mixins/ParentNode__Element.json

```diff
@@ -141,59 +95,11 @@
             "standard_track": true,
             "deprecated": false
           }
-        },
-        "document_documentfragment": {
-          "__compat": {
-            "description": "Support on <code>Document</code> and <code>DocumentFragment</code>",
-            "support": {
-              "chrome": {
-                "version_added": "29"
-              },
-              "chrome_android": {
-                "version_added": true
-              },
-              "edge": {
-                "version_added": "≤79"
-              },
-              "firefox": {
-                "version_added": "25"
-              },
-              "firefox_android": {
-                "version_added": null
-              },
-              "ie": {
-                "version_added": false
-              },
-              "opera": {
-                "version_added": "16"
-              },
-              "opera_android": {
-                "version_added": null
-              },
-              "safari": {
-                "version_added": false
-              },
-              "safari_ios": {
-                "version_added": false
-              },
-              "samsunginternet_android": {
-                "version_added": true
-              },
-              "webview_android": {
-                "version_added": "≤37"
-              }
-            },
-            "status": {
-              "experimental": false,
-              "standard_track": true,
-              "deprecated": false
-            }
-          }
         }
       },
       "children": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ParentNode/children",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/children",
```

Comment:
Things to update outside of the diff:
- safari_ios should be "3". I've tested http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8927 on Safari 4 and iOS 3 and `document.body.children` is there. It might have been supported even earlier and if this were new data I'd suggest ranged versions, but I don't want to change an existing "4" to "≤4" without evidence.